Sources/PayPal/PayPal.swift

```diff
@@ -0,0 +1,10 @@
+#if !COCOAPODS
+import PaymentsCore
+#endif
```

Comment:
We can revisit if this syntax, or `canImport()` is easier to read / makes more sense.

Reply:
This import statement is not needed (won't compile) when building for CocoaPods. It _is_ needed for building our demo app. We will test in the future whether it is also needed for SPM & revisit the best way to structure these imports then.